src/XIVLauncher.Common.Unix/Compatibility/UnixHelpers.cs

```diff
@@ -0,0 +1,96 @@
+﻿using System.IO;
+using System.Net.Http;
+using System.Threading.Tasks;
+using System.Linq;
+using System.Text.RegularExpressions;
+using Serilog;
+using XIVLauncher.Common.Util;
+
+namespace XIVLauncher.Common.Unix.Compatibility;
+
+public enum DistroPackage
```

Comment:
Maybe this can be a `LinuxDistroPackage`

Reply:
It actually shouldn't even be there. It's part of the OSInfo.cs file on the XL.Core side.